web-registry/src/pages/CreateCreditClassInfo/CreateCreditClassInfo.tsx

```diff
@@ -0,0 +1,246 @@
+import React, { useState } from 'react';
+import { makeStyles, useTheme } from '@mui/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import CardMedia from '@mui/material/CardMedia';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+
+import Section from 'web-components/lib/components/section';
+import ResponsiveSlider from 'web-components/lib/components/sliders/ResponsiveSlider';
+import FixedFooter from 'web-components/lib/components/fixed-footer';
+import ContainedButton from 'web-components/lib/components/buttons/ContainedButton';
+import Modal from 'web-components/lib/components/modal';
+import { Body, Title } from 'web-components/lib/components/typography';
+import { BlockContent } from 'web-components/lib/components/block-content';
+
+import { WrappedImpactCard, WrappedResourcesCard } from 'components/atoms';
+import { HeroTitle, HeroAction, OverviewCards } from 'components/molecules';
+import { StepCardsWithDescription } from 'components/organisms';
+
+import fernImg from 'assets/fern-in-hands.png';
+import writingOnPaperImg from 'assets/writing-on-paper.png';
+import topographyImg from 'assets/topography-pattern-cutout-1.png';
+
+import { useAllCreateCreditClassPageQuery } from 'generated/sanity-graphql';
+import { client } from 'sanity';
+
+const useStyles = makeStyles(theme => ({
```

Comment:
Should this be in `CreateCreditClassInfo.styles.tsx` ?

Reply:
ya I like that pattern overall, but didn't actually write this component - just renamed it from before (it was the prior `CreateCreditClass` page, which is info hence the rename). Went ahead and changed though 👍